### DISCUSSION TITLE
Welcome to osxphotos Discussions!

### DISCUSSION THREAD

**Discussion by RhetTbull** [RhetTbull/osxphotos#288] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about how you use osxphotos!

 This is intended as a place to ask questions, discuss use cases for osxphotos, etc. without having to open an new Issue just to have a discussion.

**Comment by synox** [comment]
Hi I'm Aravindo. I was waiting for this software since many years. I am so happy there is finally a stable solution. :-D 
Now my next project would be to setup a macOS VM where this runs every hour and exports to some permanent cloud storage. But mac vms are just soo expensive...

**Comment by RhetTbull** [comment]
Hello Aravindo -- welcome!  Glad you found osxphotos useful.  If you have any questions about how to use it, please feel free to post here.
